src/os/bluestore/BlueStore.cc

```diff
@@ -5182,10 +5149,23 @@ void BlueStore::Collection::split_cache(
       };
 
       for (auto& e : o->extent_map.extent_map) {
-	e.blob->last_encoded_id = -1;
+        e.blob->last_encoded_id = -1;
       }
       for (auto& b : o->extent_map.spanning_blob_map) {
-	b.second->last_encoded_id = -1;
+        b.second->last_encoded_id = -1;
+      }
+      // By default do not copy buffers to clones, and let them read data by
+      // themselves. The exception are 'writing' buffers, which are not yet
+      // stable on device.
+      for (auto &i : o->bc.buffer_map) {
+        if (!i.second.is_writing()) {
```

Comment:
Why cannot we also copy `writing` buffers? I was under impresion that new `finish_write` targets Onode, not blobs.